### PR TITLE
initial inspec check for sensu

### PR DESCRIPTION
**Pull request by paulczar**

### CODE REVIEW

File: sensu/plugins/check-inspec.rb [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  option :controls,
+         short: '-c /tmp/dir',
+         long: '--controls /tmp/dir',
+         required: true,
+         default: '/etc/inspec/controls'
+
+  option :severity,
+         short: '-s severity',
+         long: '--severity severity',
+         default: 'critical'
```
Comment (masteinhauser) [inline sensu/plugins/check-inspec.rb:47]: Technically, this is a deviation from the severity we use in other checks in this repo.

File: sensu/plugins/check-inspec.rb [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    msg = ""
+    results['controls'].each do |control|
+      #puts control
+      if control['status'] == "passed"
+        passed += 1
+      else
+        failed += 1
+        msg += "#{control['id']} #{control['code_desc']} - #{control['status']}\n"
+      end
+    end
```
Comment (masteinhauser) [inline sensu/plugins/check-inspec.rb:67]: 👍 Exactly how we want a single message to dump out all failing checks, not like the previous serverspec issue we had.

**Review by masteinhauser — APPROVED** [review]
It doesn't follow the standards of the repo, but it's not something I disagree with.

File: sensu/plugins/check-inspec.rb (hunk 1 above)
Comment (paulczar) [reply sensu/plugins/check-inspec.rb:47]: how?  this was an exact copy of serverspec check.

Comment (masteinhauser) [reply sensu/plugins/check-inspec.rb:47]: Ah, got it, just comparing to `-z criticality` but I think `--severity` should be fine too. Sorry for my misreading.

**Pull request merged**